modular_skyrat/modules/ashwalkers/code/effects/ash_rune.dm

```diff
@@ -0,0 +1,69 @@
+/obj/effect/ash_rune
+	name = "ash rune"
+	desc = "A remnant of a civilization that was once powerful enough to harness strange energy for transmutations."
+	icon = 'modular_skyrat/modules/ashwalkers/icons/ash_ritual.dmi'
+	icon_state = "rune"
+
+	/// the list of rituals
+	var/list/rituals = list()
+
+	/// the current chosen ritual
+	var/datum/ash_ritual/current_ritual = null
```

Comment:
get rid of this on destroy

Reply:
resistance flag says indestructible-- its not supposed to be calling destroy.

Reply:
it should be cleaning it up on destroy anyway.